torch/quantization/fx/quantization_patterns.py

```diff
@@ -288,10 +309,14 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 args = load_arg(quantized=False)(node.args)
                 kwargs = load_arg(quantized=False)(node.kwargs)
                 op_out = quantizer.quantized_graph.node_copy(node, load_arg(quantized=False))
-                cur_idx = quantizer.activation_post_process_indexes[node.name]
                 activation_post_process = \
-                    quantizer.modules[quantizer.activation_post_process_map[node.name][cur_idx]]
-                quantizer.activation_post_process_indexes[node.name] += 1
+                    self._maybe_get_last_node_only_observer(quantizer)
+                assert activation_post_process is not None
```

Comment:
should we make this assert in _maybe_get_last_node_only_observer?

Reply:
hmm, so this is possible at the individual quantize_handler class level, but not possible for the base class (because not all quantize_handlers are expected to have output be observed).

Reply:
Oh OK